src/util/factorymanager.cpp

```diff
@@ -82,9 +82,11 @@ nlohmann::json generateJsonDocumentation(const Documentation& d) {
         }
         else if (tv) {
             Documentation doc = { .entries = tv->documentations };
-            nlohmann::json restrictions = generateJsonDocumentation(doc);
             // We have a TableVerifier, so we need to recurse
```

Comment:
Should this comment stay around?  Find it a bit hard to understand what it means